app/gateways/paypal.js

```diff
@@ -15,19 +15,18 @@ const getConfig = (connectedAccount) => ({
 
 const getCallbackUrl = (group, transaction) => `${config.host.api}/groups/${group.id}/transactions/${transaction.id}/callback`;
 
-const createBillingPlan = (group, transaction, paypalConfig, cb) => {
+const createBillingPlan = (planDescription, group, transaction, paypalConfig, cb) => {
```

Comment:
Where is this called from? We need to update that call to include additional parameter.